sample/app/build.gradle

```diff
@@ -9,21 +9,21 @@ apply plugin: com.novoda.buildproperties.BuildPropertiesPlugin
 
 buildProperties {
     notThere {
-        file rootProject.file('whatever'), '''
-      This file actually doesn't exists. This is used to show properties files are lazy-loaded.
-    '''
+        from rootProject.file('whatever')
+        description =   'This file actually doesn\'t exists. This properties set is used to show how\n' +
```

Comment:
Need to drop the "s" from exists.

"This file doesn't exist.`
"This file actually doesn't exist"

"This is used to show how properties files are lazily loaded"